public/app/plugins/panel/singlestat/editor.html

```diff
@@ -61,7 +61,7 @@ <h5 class="section-heading">Coloring</h5>
     <div class="gf-form-inline">
       <div class="gf-form max-width-21">
         <label class="gf-form-label width-8">Thresholds
-          <tip>Define two threshold values&lt;br /&gt; 50,80 will produce: &lt;50 = Green, 50:80 = Yellow, &gt;80 = Red</tip>
+          <tip>Define two threshold values&lt;br /&gt; 50,80 will produce: &lt;50 = Green, 50:79 = Yellow, &gt;79 = Red</tip>
```

Comment:
This is still confusing.

How about `50,80 will produce: value < 50 = Green, 50 <= value < 80 = Yellow, value >=80 = Red`?